packages/components/psammead-program-card/src/index.jsx

```diff
@@ -0,0 +1,169 @@
+    durationColor: C_WHITE,
+  },
+  next: {
+    backgroundColor: C_WHITE,
+    headerLabelColor: C_KINGFISHER,
+    headerTextColor: C_METAL,
+    durationColor: C_KINGFISHER,
+  },
+  onDemand: {
+    backgroundColor: C_EBON,
+    headerLabelColor: '',
+    headerTextColor: C_EBON,
+    durationColor: C_WHITE,
+  },
+};
+
+const CardWrapper = styled.div`
+  padding-top: ${GEL_SPACING};
+  background-color: ${C_WHITE};
+  position: relative;
+`;
+
+const TextWrapper = styled.div`
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const HeadingWrapper = styled.h3`
+  ${({ service }) => service && getSerifRegular(service)};
+  ${({ script }) => script && getPica(script)};
+  color: ${({ headerTextColor }) => headerTextColor};
+  margin: 0; /* Reset */
+`;
+
+const LabelWrapper = styled.span`
+  ${({ service }) => service && getSansBold(service)};
+  ${({ script }) => script && getPica(script)};
+  text-transform: uppercase;
+  color: ${({ headerLabelColor }) => headerLabelColor};
+`;
+
+const DateWrapper = styled.div`
+  color: ${C_SHADOW};
+  padding: ${GEL_SPACING} 0;
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getPica(script)};
+`;
+
+const SummaryWrapper = styled(Paragraph)`
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getLongPrimer(script)};
+  color: ${C_METAL};
+  padding-bottom: ${GEL_SPACING_DBL};
+`;
+
+const ButtonWrapper = styled.div`
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getMinion(script)};
+  padding: ${GEL_SPACING} 0;
+  background-color: ${({ backgroundColor }) => backgroundColor};
+  color: ${({ durationColor }) => durationColor};
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const IconWrapper = styled.span`
+  > svg {
+    color: ${({ durationColor }) => durationColor};
+    fill: currentColor;
+    width: 1.0625rem;
+    height: ${GEL_SPACING_DBL};
+    margin: 0;
+  }
+`;
+
+const DurationWrapper = styled.span`
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const ProgramCard = ({
+  dir,
+  service,
+  script,
+  heading,
+  summary,
+  date,
+  duration,
+  state,
+  ctaLink,
+}) => (
+  <CardWrapper>
+    <Link href={ctaLink}>
+      <TextWrapper dir={dir}>
+        <HeadingWrapper
+          service={service}
+          script={script}
+          {...programStateColors[state]}
+        >
+          {state !== 'onDemand' && (
+            <LabelWrapper
+              service={service}
+              script={script}
+              {...programStateColors[state]}
+            >
+              {`${state} `}
+            </LabelWrapper>
+          )}
+          {heading}
+        </HeadingWrapper>
+        <DateWrapper service={service} script={script}>
```

Comment:
Sorry it's not particularly clear from the designs but this might not be a date! It's the title of the episode, it just happens that a lot of the examples are a date.

Reply:
Noted, I'll check with UX to get some designs that are not entirely dates.

Reply:
Currently, we do not have designs that show an episode title not in a date format. We've made reservations for this by adding necessary padding to accommodate a multiline episode title

Reply:
since this value might not be a date, is it worth changing it to like `subHeading`?

Reply:
We've already changed this to `TitleWrapper` which can span multiple lines.

Reply:
ah yes, I see that now. nice one 💯